README.rst

```diff
@@ -308,6 +308,112 @@ In case you need to run additional instructions just before the build step you c
 
 You can find more patches in the `patch catalog <#template-patch-catalog>`_ below.
 
+Using Frontend Plugin Slots
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It's possible to take advantage of this plugin's hooks to configure frontend plugin slots. Let's say you want to replace the entire footer with a simple message. Where before you might have had to fork ``frontend-component-footer``, the following is all that's currently needed:
+
+.. code-block:: python
+
+    from tutormfe.hooks import PLUGIN_SLOTS
+
+    PLUGIN_SLOTS.add_item((
+        "all", "footer_slot",
+        """
+          {
+            /* Hide the default footer. */
+            op: PLUGIN_OPERATIONS.Hide,
+            widgetId: 'default_contents',
+          },
```

Comment:
This is a bit of a nitpick: I suggest having separate items for every plugin slot. I.e:

```
 PLUGIN_SLOTS.add_items(
    ("all", "footer_slot",
        """
          {
            /* Hide the default footer. */
            op: PLUGIN_OPERATIONS.Hide,
            widgetId: 'default_contents'
          }),
    ("all", "footer_slot", """..."""),
    ...
)
```

In particular, with this approach we don't have to be too careful with trailing commas.

This is a minor concern and should not be blocking this PR.